__tests__/patch.js

```diff
@@ -87,7 +87,7 @@ describe("applyPatches", () => {
         expect(() => {
             const patch = {op: "remove", path: [0]}
             applyPatches([1, 2], [patch])
-        }).toThrowError(/^Only the last index of an array can be removed/)
+        }).not.toThrowError()
```

Comment:
Remove both `expect(() => {` and `}).not.toThrowError()`.

Test that the result of `applyPatches` equals `[2]`.

And remove both comments in this test.